### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kilrah**

### CODE REVIEW

File: radio/src/targets/horus/board.h [hunk 1]
```diff
@@ -506,12 +506,14 @@ void hapticOff(void);
 void hapticOn(uint32_t pwmPercent);
 
 // GPS driver
+#if defined(PCBX12S) && defined(INTERNAL_GPS)
```
Comment (bsongis) [inline radio/src/targets/horus/board.h:509]: it shouldn't be needed to test PCBX12S here

Comment (bsongis) [reply radio/src/targets/horus/board.h:509]: do you need some help?

**Comment by projectkk2glider** [comment]
@kilrah what is up with this PR?

**Comment by kilrah** [comment]
I don't even remember... but I think this was intended to fix a combination that didn't build. Oh well, will fix if it ever pops up again.

**Pull request closed without merging**